jenkins.py

```diff
@@ -104,8 +105,11 @@ def _api_call(url, opener, http_timeout):
     parsed_url = urlparse.urlparse(url)
     url = '{0}://{1}{2}'.format(parsed_url.scheme, parsed_url.netloc, urllib_quote(parsed_url.path))
     try:
+        ctx = ssl.create_default_context()
+        ctx.check_hostname = False
+        ctx.verify_mode = ssl.CERT_NONE
         urllib2.install_opener(opener)
-        resp = urllib2.urlopen(url, timeout=http_timeout)
+        resp = urllib2.urlopen(url, timeout=http_timeout, context=ctx)
```

Comment:
ctx is not supported in python 2.6 which is present in some of Amazon's Linux.

Reply:
:)
this snippets are my **first** step in python ...